release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2461,6 +2475,14 @@ If NMEA sentences are lost on their way to the e810 NIC, the T-GM cannot synchro
 A proposed fix is to report a `FREERUN` event when the NMEA string is lost.
 (link:https://issues.redhat.com/browse/OCPBUGS-19838[*OCPBUGS-19838*])
 
+* If you have IPsec enabled on the cluster, a restart of the `ipsec.service` on the underlying node or deletion of the NMState Operator `NodeNetworkConfigurationPolicy` object configuration for the IPsec tunnel causes a loss of the IPsec configuration for encryption of external traffic. (link:https://issues.redhat.com/browse/RHELPLAN-169398[*RHELPLAN-169398*])
```

Comment:
Not `or deletion of the NMState Operator `NodeNetworkConfigurationPolicy` object configuration for the IPsec tunnel`  but `or restarting the ovn-ipsec-host pod on same node NS is established`

Reply:
@huiran0826, revised to:

> If you have IPsec enabled on the cluster, on the node hosting the north-south IPsec connection, restarting the `ipsec.service` systemd unit or restarting the `ovn-ipsec-host` pod causes a loss of the IPsec connection.